possible missing components in Vulkan libraries?

<!--
## Before you submit an issue
Please look up our issues, maybe your problem was mentioned before.
-->

## Description
<!-- Provide a short description of your problem here. -->
I've been trying to build Vulkan-Tools following [their GitHub repo](https://github.com/KhronosGroup/Vulkan-Tools/blob/master/BUILD.md), and it only seems to work when I *don't* run their included [script](https://github.com/KhronosGroup/Vulkan-Tools/blob/master/scripts/update_deps.py) to fetch dependencies directly from the GitHub repo (pretty much just Vulkan headers and the Vulkan loader, [from what I can tell](https://github.com/KhronosGroup/Vulkan-Tools/blob/master/scripts/known_good.json)).

If you executed a specific command:
```shell
cd /usr/local/mystuff
https://github.com/KhronosGroup/Vulkan-Tools --depth=1
cd Vulkan-Tools
git pull
mkdir build
cd build
# ../scripts/update_deps.py
cmake ..
make -j$(nproc)
```

Output:
```shell
chronos@localhost /usr/local/mystuff/Vulkan-Tools/build $ cmake .. && make -j$(nproc)
-- The C compiler identification is GNU 12.2.0
-- The CXX compiler identification is GNU 12.2.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/local/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/local/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Using find_package to locate Vulkan
-- Found Vulkan: /usr/local/lib64/libvulkan.so  
-- Found VulkanHeaders: /usr/local/include  
-- Found VulkanRegistry: /usr/local/share/vulkan/registry  
-- Vulkan FOUND = TRUE
-- Vulkan Lib Dir = /usr/local/lib64
-- Vulkan Lib = /usr/local/lib64/libvulkan.so
-- Vulkan Headers Include = /usr/local/include
-- Vulkan Headers Registry = /usr/local/share/vulkan/registry
-- Using cmake find_program to look for glslangValidator
   Using glslangValidator at /usr/local/mystuff/Vulkan-Tools/scripts/../glslang/linux/bin
-- Found PkgConfig: /usr/local/bin/pkg-config (found version "0.29.2") 
-- Found XCB: /usr/local/include  
-- Found X11: /usr/local/include   
-- Looking for XOpenDisplay in /usr/local/lib64/libX11.so;/usr/local/lib64/libXext.so
-- Looking for XOpenDisplay in /usr/local/lib64/libX11.so;/usr/local/lib64/libXext.so - found
-- Looking for gethostbyname
-- Looking for gethostbyname - found
-- Looking for connect
-- Looking for connect - found
-- Looking for remove
-- Looking for remove - found
-- Looking for shmat
-- Looking for shmat - found
-- Looking for IceConnectionNumber in ICE
-- Looking for IceConnectionNumber in ICE - found
CMake Warning (dev) at /usr/local/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args`
  (WAYLAND_CLIENT) does not match the name of the calling package (Wayland).
  This can lead to problems in calling code that expects `find_package`
  result variables (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindWayland.cmake:52 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  cube/CMakeLists.txt:68 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found WAYLAND_CLIENT: /usr/local/lib64/libwayland-client.so  
CMake Warning (dev) at /usr/local/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args`
  (WAYLAND_SERVER) does not match the name of the calling package (Wayland).
  This can lead to problems in calling code that expects `find_package`
  result variables (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindWayland.cmake:53 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  cube/CMakeLists.txt:68 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found WAYLAND_SERVER: /usr/local/lib64/libwayland-server.so  
CMake Warning (dev) at /usr/local/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args`
  (WAYLAND_EGL) does not match the name of the calling package (Wayland).
  This can lead to problems in calling code that expects `find_package`
  result variables (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindWayland.cmake:54 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  cube/CMakeLists.txt:68 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found WAYLAND_EGL: /usr/local/lib64/libwayland-egl.so  
CMake Warning (dev) at /usr/local/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args`
  (WAYLAND_CURSOR) does not match the name of the calling package (Wayland).
  This can lead to problems in calling code that expects `find_package`
  result variables (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindWayland.cmake:55 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  cube/CMakeLists.txt:68 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found WAYLAND_CURSOR: /usr/local/lib64/libwayland-cursor.so  
CMake Warning (dev) at /usr/local/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args` (WAYLAND)
  does not match the name of the calling package (Wayland).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindWayland.cmake:56 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  cube/CMakeLists.txt:68 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found WAYLAND: /usr/local/lib64/libwayland-client.so;/usr/local/lib64/libwayland-server.so;/usr/local/lib64/libwayland-egl.so;/usr/local/lib64/libwayland-cursor.so  
CMake Warning (dev) at /usr/local/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args` (WAYLAND)
  does not match the name of the calling package (WaylandScanner).  This can
  lead to problems in calling code that expects `find_package` result
  variables (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindWaylandScanner.cmake:12 (find_package_handle_standard_args)
  cube/CMakeLists.txt:69 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found WAYLAND: /usr/local/bin/wayland-scanner  
-- Checking for module 'wayland-protocols'
--   Found wayland-protocols, version 1.26
CMake Warning (dev) at /usr/local/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args` (WAYLAND)
  does not match the name of the calling package (WaylandProtocols).  This
  can lead to problems in calling code that expects `find_package` result
  variables (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindWaylandProtocols.cmake:12 (find_package_handle_standard_args)
  cube/CMakeLists.txt:70 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found WAYLAND: //usr/local/share/wayland-protocols  
-- Looking for clock_gettime in rt
-- Looking for clock_gettime in rt - found
CMake Warning (dev) at /usr/local/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args`
  (WAYLAND_CLIENT) does not match the name of the calling package (Wayland).
  This can lead to problems in calling code that expects `find_package`
  result variables (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindWayland.cmake:52 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  vulkaninfo/CMakeLists.txt:85 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at /usr/local/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args`
  (WAYLAND_SERVER) does not match the name of the calling package (Wayland).
  This can lead to problems in calling code that expects `find_package`
  result variables (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindWayland.cmake:53 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  vulkaninfo/CMakeLists.txt:85 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at /usr/local/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args`
  (WAYLAND_EGL) does not match the name of the calling package (Wayland).
  This can lead to problems in calling code that expects `find_package`
  result variables (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindWayland.cmake:54 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  vulkaninfo/CMakeLists.txt:85 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at /usr/local/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args`
  (WAYLAND_CURSOR) does not match the name of the calling package (Wayland).
  This can lead to problems in calling code that expects `find_package`
  result variables (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindWayland.cmake:55 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  vulkaninfo/CMakeLists.txt:85 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Warning (dev) at /usr/local/share/cmake-3.24/Modules/FindPackageHandleStandardArgs.cmake:438 (message):
  The package name passed to `find_package_handle_standard_args` (WAYLAND)
  does not match the name of the calling package (Wayland).  This can lead to
  problems in calling code that expects `find_package` result variables
  (e.g., `_FOUND`) to follow a certain pattern.
Call Stack (most recent call first):
  cmake/FindWayland.cmake:56 (FIND_PACKAGE_HANDLE_STANDARD_ARGS)
  vulkaninfo/CMakeLists.txt:85 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found WAYLAND: /usr/local/lib64/libwayland-client.so;/usr/local/lib64/libwayland-server.so;/usr/local/lib64/libwayland-egl.so;/usr/local/lib64/libwayland-cursor.so  
-- Configuring done
-- Generating done
-- Build files have been written to: /usr/local/mystuff/Vulkan-Tools/build
[  4%] Compiling cube vertex shader



#normal build progress stuff



[ 81%] Building CXX object vulkaninfo/CMakeFiles/vulkaninfo.dir/vulkaninfo.cpp.o
In file included from /usr/local/mystuff/Vulkan-Tools/vulkaninfo/vulkaninfo.cpp:33:
/usr/local/mystuff/Vulkan-Tools/vulkaninfo/generated/vulkaninfo.hpp: In function ‘std::string VkFormatString(VkFormat)’:
/usr/local/mystuff/Vulkan-Tools/vulkaninfo/generated/vulkaninfo.hpp:357:15: error: ‘VK_FORMAT_R16G16_S10_5_NV’ was not declared in this scope; did you mean ‘VK_FORMAT_R16G16_SINT’?
  357 |         case (VK_FORMAT_R16G16_S10_5_NV): return "FORMAT_R16G16_S10_5_NV";
      |               ^~~~~~~~~~~~~~~~~~~~~~~~~
      |               VK_FORMAT_R16G16_SINT
/usr/local/mystuff/Vulkan-Tools/vulkaninfo/generated/vulkaninfo.hpp: In function ‘std::vector<const char*> VkFormatFeatureFlagBits2GetStrings(VkFormatFeatureFlagBits2)’:



#many more errors that I had to cut out of the output because GitHub said the issue was too long



/usr/local/mystuff/Vulkan-Tools/icd/generated/mock_icd.cpp:5990:11: error: ‘VkColorBlendAdvancedEXT’ does not name a type
 5990 |     const VkColorBlendAdvancedEXT*              pColorBlendAdvanced)
      |           ^~~~~~~~~~~~~~~~~~~~~~~
/usr/local/mystuff/Vulkan-Tools/icd/generated/mock_icd.cpp:6118:11: error: ‘VkOpticalFlowImageFormatInfoNV’ does not name a type
 6118 |     const VkOpticalFlowImageFormatInfoNV*       pOpticalFlowImageFormatInfo,
      |           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/mystuff/Vulkan-Tools/icd/generated/mock_icd.cpp:6120:5: error: ‘VkOpticalFlowImageFormatPropertiesNV’ has not been declared
 6120 |     VkOpticalFlowImageFormatPropertiesNV*       pImageFormatProperties)
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/mystuff/Vulkan-Tools/icd/generated/mock_icd.cpp:6128:11: error: ‘VkOpticalFlowSessionCreateInfoNV’ does not name a type
 6128 |     const VkOpticalFlowSessionCreateInfoNV*     pCreateInfo,
      |           ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/mystuff/Vulkan-Tools/icd/generated/mock_icd.cpp:6130:5: error: ‘VkOpticalFlowSessionNV’ has not been declared
 6130 |     VkOpticalFlowSessionNV*                     pSession)
      |     ^~~~~~~~~~~~~~~~~~~~~~
/usr/local/mystuff/Vulkan-Tools/icd/generated/mock_icd.cpp: In function ‘VkResult vkmock::CreateOpticalFlowSessionNV(VkDevice, const int*, const VkAllocationCallbacks*, int*)’:
/usr/local/mystuff/Vulkan-Tools/icd/generated/mock_icd.cpp:6133:18: error: ‘VkOpticalFlowSessionNV’ was not declared in this scope; did you mean ‘CreateOpticalFlowSessionNV’?
 6133 |     *pSession = (VkOpticalFlowSessionNV)global_unique_handle++;
      |                  ^~~~~~~~~~~~~~~~~~~~~~
      |                  CreateOpticalFlowSessionNV
/usr/local/mystuff/Vulkan-Tools/icd/generated/mock_icd.cpp: At global scope:
/usr/local/mystuff/Vulkan-Tools/icd/generated/mock_icd.cpp:6139:5: error: ‘VkOpticalFlowSessionNV’ has not been declared
 6139 |     VkOpticalFlowSessionNV                      session,
      |     ^~~~~~~~~~~~~~~~~~~~~~
/usr/local/mystuff/Vulkan-Tools/icd/generated/mock_icd.cpp:6147:5: error: ‘VkOpticalFlowSessionNV’ has not been declared
 6147 |     VkOpticalFlowSessionNV                      session,
      |     ^~~~~~~~~~~~~~~~~~~~~~
/usr/local/mystuff/Vulkan-Tools/icd/generated/mock_icd.cpp:6148:5: error: ‘VkOpticalFlowSessionBindingPointNV’ has not been declared
 6148 |     VkOpticalFlowSessionBindingPointNV          bindingPoint,
      |     ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/usr/local/mystuff/Vulkan-Tools/icd/generated/mock_icd.cpp:6158:5: error: ‘VkOpticalFlowSessionNV’ has not been declared
 6158 |     VkOpticalFlowSessionNV                      session,
      |     ^~~~~~~~~~~~~~~~~~~~~~
/usr/local/mystuff/Vulkan-Tools/icd/generated/mock_icd.cpp:6159:11: error: ‘VkOpticalFlowExecuteInfoNV’ does not name a type
 6159 |     const VkOpticalFlowExecuteInfoNV*           pExecuteInfo)
      |           ^~~~~~~~~~~~~~~~~~~~~~~~~~
make[2]: *** [icd/CMakeFiles/VkICD_mock_icd.dir/build.make:76: icd/CMakeFiles/VkICD_mock_icd.dir/generated/mock_icd.cpp.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:351: icd/CMakeFiles/VkICD_mock_icd.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

## Versions and system information
<!-- 
Paste the output of `crew sysinfo` between two `***` below

*If `crew sysinfo` does not work, use `cat /etc/lsb-release` instead.
-->
***
Ignoring debug-1.4.0 because its extensions are not built. Try: gem pristine debug --version 1.4.0
- Architecture: `x86_64` (`x86_64`)
- Kernel version: `4.19.241-13212-ge9b0bab46475`

- Chromebrew version: `1.25.4`
- Chromebrew prefix: `/usr/local`
- Chromebrew libdir: `/usr/local/lib64`

- Last update in local repository: b5e4ced `Re-enable And/Or cop, lint tree (#7457) (28 hours ago)`

- OS variant: `Chrome OS`
- OS version: `celes-release/R103-14816.131.0`
- OS channel: `stable-channel`
***

## Additional information
<!--
Mention things we might need to know. Like:

What I think needs to be done:
- [ ] step one
- [ ] step two
-->
As previously mentioned, running the commented-out line in my example to fetch Vulkan deps directly from their GitHub repositories works perfectly (and I can even get a `vkcube` going _without sommelier!_)
I propose changing the [vulkan_headers](https://github.com/KhronosGroup/Vulkan-Headers/tags) and [vulkan_icd_loader](https://github.com/KhronosGroup/Vulkan-Loader/tags) packages to use the tarballs directly from the tagged releases on GitHub rather than a custom build - although, full disclosure, I haven't had the time to test whether or not this would fix the issue I'm having.
<!--
## That's it
Thank you for submitting your issue.
When done, please delete the parts of this template which you don't need or these, which are only for guidance.
-->

--- a/packages/vulkan_headers.rb
+++ b/packages/vulkan_headers.rb
@@ -3,25 +3,26 @@ require 'package'
 class Vulkan_headers < Package
   description 'Vulkan header files'
   homepage 'https://github.com/KhronosGroup/Vulkan-Headers'
-  @_ver = '1.3.224'
+  @_ver = '1.3.230'
   version @_ver
   license 'Apache-2.0'
   compatibility 'all'
   source_url 'https://github.com/KhronosGroup/Vulkan-Headers.git'
-  git_hashtag "v#{@_ver}"
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_headers/1.3.224_armv7l/vulkan_headers-1.3.224-chromeos-armv7l.tar.zst',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_headers/1.3.224_armv7l/vulkan_headers-1.3.224-chromeos-armv7l.tar.zst',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_headers/1.3.224_i686/vulkan_headers-1.3.224-chromeos-i686.tar.zst',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_headers/1.3.224_x86_64/vulkan_headers-1.3.224-chromeos-x86_64.tar.zst'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_headers/1.3.230_armv7l/vulkan_headers-1.3.230-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_headers/1.3.230_armv7l/vulkan_headers-1.3.230-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_headers/1.3.230_i686/vulkan_headers-1.3.230-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_headers/1.3.230_x86_64/vulkan_headers-1.3.230-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '07d06412605ad77da5eb584ff0004ca8bd375facdec7f208cc8507d87574770a',
-     armv7l: '07d06412605ad77da5eb584ff0004ca8bd375facdec7f208cc8507d87574770a',
-       i686: '09b838eacafe97c1471f3da15a82fef46bbd895b443f50f0cefb080b4e513595',
-     x86_64: '20e3b90d93447e620d730bfe3f0f55d97cfd19134ed275b5672f5d962863118a'
+    aarch64: 'd36b9887edca21f8092ed048bdce3b097312e1074c34a801088ac8eeadb9981b',
+     armv7l: 'd36b9887edca21f8092ed048bdce3b097312e1074c34a801088ac8eeadb9981b',
+       i686: 'de184f007aad9a8b937c769e9cbd37a4257e7d7bde3e39eaa947f0b7ee1d5369',
+     x86_64: 'db133f28b049375d584831c05020bb25778a7728d12d5f27198ee3d3f1c56d87'
   })
+
+  git_hashtag "v#{@_ver}"
 
   def self.build
     Dir.mkdir 'builddir'
@@ -31,10 +32,10 @@ class Vulkan_headers < Package
         #{CREW_CMAKE_OPTIONS} \
         .."
     end
-    system 'ninja -C builddir'
+    system 'mold -run samu -C builddir'
   end
 
   def self.install
-    system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
+    system "DESTDIR=#{CREW_DEST_DIR} samu -C builddir install"
   end
 end

--- a/packages/vulkan_icd_loader.rb
+++ b/packages/vulkan_icd_loader.rb
@@ -3,25 +3,26 @@ require 'package'
 class Vulkan_icd_loader < Package
   description 'Vulkan Installable Client Driver ICD Loader'
   homepage 'https://github.com/KhronosGroup/Vulkan-Loader'
-  @_ver = '1.3.224'
+  @_ver = '1.3.230'
   version @_ver
   license 'Apache-2.0'
   compatibility 'all'
   source_url 'https://github.com/KhronosGroup/Vulkan-Loader.git'
-  git_hashtag "v#{@_ver}"
 
   binary_url({
-    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_icd_loader/1.3.224_armv7l/vulkan_icd_loader-1.3.224-chromeos-armv7l.tar.zst',
-     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_icd_loader/1.3.224_armv7l/vulkan_icd_loader-1.3.224-chromeos-armv7l.tar.zst',
-       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_icd_loader/1.3.224_i686/vulkan_icd_loader-1.3.224-chromeos-i686.tar.zst',
-     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_icd_loader/1.3.224_x86_64/vulkan_icd_loader-1.3.224-chromeos-x86_64.tar.zst'
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_icd_loader/1.3.230_armv7l/vulkan_icd_loader-1.3.230-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_icd_loader/1.3.230_armv7l/vulkan_icd_loader-1.3.230-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_icd_loader/1.3.230_i686/vulkan_icd_loader-1.3.230-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_icd_loader/1.3.230_x86_64/vulkan_icd_loader-1.3.230-chromeos-x86_64.tar.zst'
   })
   binary_sha256({
-    aarch64: '15b9c5272c0a2a8adcafd5984da16738a3a7032479b38f9ce77a3b4ddaf187e5',
-     armv7l: '15b9c5272c0a2a8adcafd5984da16738a3a7032479b38f9ce77a3b4ddaf187e5',
-       i686: 'c0c0c5a6fed09d2515de58c8f2af31081133fe8bc292d445b7bf5ce8676a4239',
-     x86_64: '53fdaa92fb1e31410c6ebd50f1661f8b99b3a62a2f1bc33be123b2fff7a3f2da'
+    aarch64: '15f8878ef502ebe90ed22535d2e99e6da804e02aae41f0a4c7403c4ae512b26f',
+     armv7l: '15f8878ef502ebe90ed22535d2e99e6da804e02aae41f0a4c7403c4ae512b26f',
+       i686: 'f36cd322d98a6c6874e9c450cf5bbde693b297cd8315f6e01d7cc94f6ff2d982',
+     x86_64: 'fdfb0631b3069712a4aa0095cd0055d5425d842a76bfa5e979fdba04a6a49f7e'
   })
+
+  git_hashtag "v#{@_ver}"
 
   depends_on 'libx11'
   depends_on 'libxrandr'

--- a/packages/vulkan_tools.rb
+++ b/packages/vulkan_tools.rb
@@ -1,0 +1,81 @@
+# Adapted from Arch Linux vulkan-tools PKGBUILD at:
+# https://github.com/archlinux/svntogit-packages/raw/packages/vulkan-tools/trunk/PKGBUILD
+
+require 'package'
+
+class Vulkan_tools < Package
+  description 'Vulkan Utilities and Tools'
+  homepage 'https://www.khronos.org/vulkan/'
+  version '1.3.230'
+  license 'custom'
+  compatibility 'all'
+  source_url "https://github.com/KhronosGroup/Vulkan-Tools/archive/v#{version}.tar.gz"
+  source_sha256 '5bb190d20ee8ae4e8dd157b686bd2d3162dc3a814b3d0625d90b1d84dd177dbb'
+
+  binary_url({
+    aarch64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_tools/1.3.230_armv7l/vulkan_tools-1.3.230-chromeos-armv7l.tar.zst',
+     armv7l: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_tools/1.3.230_armv7l/vulkan_tools-1.3.230-chromeos-armv7l.tar.zst',
+       i686: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_tools/1.3.230_i686/vulkan_tools-1.3.230-chromeos-i686.tar.zst',
+     x86_64: 'https://gitlab.com/api/v4/projects/26210301/packages/generic/vulkan_tools/1.3.230_x86_64/vulkan_tools-1.3.230-chromeos-x86_64.tar.zst'
+  })
+  binary_sha256({
+    aarch64: 'be40fe9684ff980a1e4624ec112eba844b6a58e219cf71bb6bb4f1a75667693e',
+     armv7l: 'be40fe9684ff980a1e4624ec112eba844b6a58e219cf71bb6bb4f1a75667693e',
+       i686: '428aadff66735013c5dfc3b392cf54055b6d9dba0f710492fe4efd1c077e3674',
+     x86_64: 'fbe30a5c9296eaaaab392540792bbf729166979cc5313b2d306923fe3ec3c437'
+  })
+
+  depends_on 'libx11'
+  depends_on 'wayland'
+  depends_on 'python3' => :build
+  depends_on 'vulkan_headers' => :build
+  depends_on 'vulkan_icd_loader' => :build
+  depends_on 'wayland_protocols' => :build
+  depends_on 'glslang' => :build
+  depends_on 'spirv_tools' => :build
+
+  def self.build
+    Dir.mkdir 'builddir'
+    Dir.chdir 'builddir' do
+      system "env #{CREW_ENV_OPTIONS} \
+        cmake -G Ninja \
+        #{CREW_CMAKE_OPTIONS} \
+        -DVULKAN_HEADERS_INSTALL_DIR=#{CREW_PREFIX} \
+        -DCMAKE_INSTALL_SYSCONFDIR=#{CREW_PREFIX}/etc \
+        -DCMAKE_INSTALL_DATADIR=#{CREW_PREFIX}/share \
+        -DCMAKE_SKIP_RPATH=True \
+        -DBUILD_WSI_XCB_SUPPORT=On \
+        -DBUILD_WSI_XLIB_SUPPORT=On \
+        -DBUILD_WSI_WAYLAND_SUPPORT=On \
+        -DBUILD_CUBE=ON \
+        -DBUILD_VULKANINFO=ON \
+        -DBUILD_ICD=OFF \
+        .."
+    end
+    Dir.mkdir 'builddir-wayland'
+    Dir.chdir 'builddir-wayland' do
+      system "env #{CREW_ENV_OPTIONS} \
+        cmake -G Ninja \
+        #{CREW_CMAKE_OPTIONS} \
+        -DVULKAN_HEADERS_INSTALL_DIR=#{CREW_PREFIX} \
+        -DCMAKE_INSTALL_SYSCONFDIR=#{CREW_PREFIX}/etc \
+        -DCMAKE_INSTALL_DATADIR=#{CREW_PREFIX}/share \
+        -DCMAKE_SKIP_RPATH=True \
+        -DBUILD_WSI_XCB_SUPPORT=OFF \
+        -DBUILD_WSI_XLIB_SUPPORT=OFF \
+        -DBUILD_WSI_WAYLAND_SUPPORT=On \
+        -DBUILD_CUBE=ON \
+        -DCUBE_WSI_SELECTION=WAYLAND \
+        -DBUILD_VULKANINFO=OFF \
+        -DBUILD_ICD=OFF \
+        .."
+    end
+    system 'samu -C builddir'
+    system 'samu -C builddir-wayland'
+  end
+
+  def self.install
+    system "DESTDIR=#{CREW_DEST_DIR} samu -C builddir install"
+    FileUtils.install 'builddir-wayland/cube/vkcube-wayland', "#{CREW_DEST_PREFIX}/bin/vkcube-wayland", mode: 0o755
+  end
+end

--- a/tools/packages.yaml
+++ b/tools/packages.yaml
@@ -7750,12 +7750,17 @@ activity: medium
 ---
 kind: url
 name: vulkan_headers
-url: https://github.com/KhronosGroup/Vulkan-Headers/releases
+url: https://github.com/KhronosGroup/Vulkan-Headers/tags
 activity: high
 ---
 kind: url
 name: vulkan_icd_loader
-url: https://github.com/KhronosGroup/Vulkan-Loader/releases
+url: https://github.com/KhronosGroup/Vulkan-Loader/tags
+activity: high
+---
+kind: url
+name: vulkan_tools
+url: https://github.com/KhronosGroup/Vulkan-Tools/tags
 activity: high
 ---
 kind: url

